MODULE.bazel

```diff
@@ -0,0 +1,197 @@
+
+# Dependencies:
+bazel_dep(name = "abseil-cpp", version = "20240116.1", repo_name = "com_google_absl")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1", repo_name = "com_google_googletest")
+bazel_dep(name = "re2", version = "2024-02-01", repo_name = "com_googlesource_code_re2")
+bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "rules_cc", version = "0.1.2")
+bazel_dep(name = "rules_fuzzing", version = "0.5.2")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_proto", version = "6.0.2")
+bazel_dep(name = "aspect_rules_lint", version = "1.0.8")
+bazel_dep(name = "rules_rust", version = "0.59.2")
+
+# Protobuf matchers for googletest.
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_protobuf_matchers",
+    build_file_content = 'package(default_visibility = ["//visibility:public"])',
+    patch_args = ["-p1"],
+    patches = ["//third_party/google:protobuf_matchers_fix.patch"],
+    sha256 = "8314521014fb7b5e33f061d0f53a3c7222dbee1871df2f66198522a5687a71c1",
+    strip_prefix = "protobuf-matchers-7c8e15741bcea83db7819cc472c3e96301a95158",
+    urls = ["https://github.com/inazarenko/protobuf-matchers/archive/7c8e15741bcea83db7819cc472c3e96301a95158.zip"],
+)
+
+# Use a modern gRPC (compatible with Protobuf 27+)
+bazel_dep(name = "grpc", version = "1.66.0.bcr.3", repo_name = "com_github_grpc_grpc")
+single_version_override(
+    module_name = "grpc",
+    patch_strip = 0,
+    patches = [
+        "third_party/google/grpc_windows_config_setting.patch",
+        "third_party/google/grpc_windows_endpoint_fix.patch",
+    ],
+)
+
+# Pin Protobuf explicitly to ensure sync
+bazel_dep(name = "protobuf", version = "27.3", repo_name = "com_google_protobuf")
+
+# Explicitly add upb (required by gRPC 1.66+ in Bzlmod)
+bazel_dep(name = "upb", version = "0.0.0-20230907-e7430e6")
+bazel_dep(name = "rules_foreign_cc", version = "0.9.0")
+single_version_override(
+    module_name = "rules_foreign_cc",
+    version = "0.9.0",
+)
+
+bazel_dep(name = "rules_apple", version = "3.5.1", repo_name = "build_bazel_rules_apple")
+bazel_dep(name = "rules_swift", version = "1.18.0", repo_name = "build_bazel_rules_swift")
+
+crate = use_extension("@rules_rust//crate_universe:extension.bzl", "crate")
+crate.from_cargo(
+    name = "provisioning_index",
+    cargo_lockfile = "@lowrisc_opentitan//third_party/rust:Cargo.lock",
+    manifests = ["@lowrisc_opentitan//third_party/rust:Cargo.toml"],
+    supported_platform_triples = ["x86_64-unknown-linux-gnu"],
+)
+use_repo(crate, crate_index = "provisioning_index")
+
+# Dev dependencies:
+bazel_dep(
+    name = "buildifier_prebuilt",
+    version = "6.4.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "lowrisc_misc_linters",
+    dev_dependency = True,
+)
+
+bazel_dep(
+    name = "lowrisc_opentitan",
+)
+
+archive_override(
```

Comment:
nit: should we group this next to the bazel_dep it is associated with?

or is this the result of buildifier autoformatting?